rest/client.go

```diff
@@ -79,3 +81,18 @@ func (c *client) DeleteNodeData(ctx context.Context, nodeID uuid.UUID, deleteNod
 
 	return nil
 }
+
+func (c *client) GetBandOverall(ctx context.Context, measurementID uuid.UUID, startFrequency, stopFrequency int64) (models.ModelNodeBandOverallResponse, error) {
```

Comment:
Do we also want to add support for using `windowFunction` and `detectionType` here?